examples/person-detection/person_detection.py

```diff
@@ -23,12 +24,35 @@
 !mkdir -p binaries/person_detection && cd binaries/person_detection && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/person_detection/person_detection.ino.elf # fetch prebuilt binaries
 
 # %% [markdown]
-"""## Take a photo"""
+"""
+## Take a photo
+To change option rerun the cell
+# """
 
 # %%
+import ipywidgets as w
 from IPython.display import Image
-photo = take_photo()
-display(Image(photo))
+
+def handle_selection(obj):
+  if obj['new'] == 'Camera':
+    photo = take_photo()
+    display(Image(photo))
+  elif obj['new'] == 'Upload':
+    display(uploadButton)
+    uploadButton.observe(handle_upload, 'data')
+  toggle.layout.display = 'none'
+
+def handle_upload(obj):
+  uploader = obj['owner']
+  with open('photo.jpg', 'wb') as f:
+        f.write(uploader.data[0])
+  uploadButton.layout.display = 'none'
+  display(Image('photo.jpg'))
+
+uploadButton = w.FileUpload(accept='.jpg')
+toggle = w.ToggleButtons(options=['Default', 'Camera', 'Upload'])
+display(toggle)
+toggle.observe(handle_selection, 'value')
```

Comment:
The same here - shouldn't `toggle` be configured before displaying it?

Reply:
line 53?